CHANGELOG.md

```diff
@@ -4,8 +4,11 @@
 
 ## What's Changed
 
-## Breaking changes
+### Breaking Changes
+
 * Removed `mnemonic.to_public_key` in favor of `account.address_from_private_key`.
+* Removed `future` package in favor of `transaction` pacakge.
```

Comment:
@ahangsu Optional _suggestion_  - I think it'd be clearer to fully qualify the referenced package.  

If you keep as is, there is a minor typo:  s/pacakge/package/.

```suggestion
* Removed `algosdk.future` package.  Moved package contents to `algosdk`.
```

Reply:
that makes sense to me, thanks for the suggestion!